packages/api-client/src/conversation/Conversation.ts

```diff
@@ -109,3 +109,10 @@ export interface Conversation {
 
   protocol: ConversationProtocol;
 }
+
+export interface MLSConversation extends Conversation {
+  group_id: string;
+  epoch: number;
+  cipher_suite: number;
+  protocol: ConversationProtocol.MLS;
+}
```

Comment:
We are sure that methods like `getMLS1to1Conversations` will return an mls conversation, we can extend the type to tell typescript that groupid and epoch are defined in this case.